tests/functional/metadata/mongodb/withCond.spec.js

```diff
@@ -16,7 +16,7 @@ const bucketName = 'testbucket';
 const mongoserver = new MongoMemoryReplSet({
     debug: false,
     instanceOpts: [
-        { port: 27018 },
+        { port: 27022 },
```

Comment:
What's the issue with having the same port everywhere? You could put a little bit more details on the rationale behind this change in the commit message :slightly_smiling_face:

Reply:
CI fails when i use the same port everywhere (with the error can't connect to mongodb, connection refused), and when i checked the logs i noticed that test outputs overlap each other which left me thinking that maybe the tests are ran in parallel, not sure it is the case though

Reply:
Could try initializing `mongoserver` inside the `beforeAll`. If that doesn't work, we can merge as is - this is a pattern that's present in a lot of places in this code IIRC so it can be taken care of by a refactor pass at some point.